src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball randomBall = lottery.getRandomBall();
+            StringBuilder nameBall = new StringBuilder(randomBall.getColour());
+            System.out.println(nameBall.append(" ").append(randomBall.getNumber()));
```

Comment:
just a recommendation for future
You can override method `toString()` in your classes, and then when you pass your object to `System.out.println(ball)` - returned value from `toString()` will be in console